libraries/Microsoft.Bot.Connector/Authentication/ChannelValidation.cs

```diff
@@ -12,17 +12,14 @@ namespace Microsoft.Bot.Connector.Authentication
 {
     public static class ChannelValidation
     {
-        // This claim is ONLY used in the Channel Validation, and not in the emulator validation        
-        private const string ServiceUrlClaim = "serviceurl";
-
         /// <summary>
         /// TO BOT FROM CHANNEL: Token validation parameters when connecting to a bot
         /// </summary>
         public static readonly TokenValidationParameters ToBotFromChannelTokenValidationParameters =
             new TokenValidationParameters()
             {
                 ValidateIssuer = true,
-                ValidIssuers = new[] { AuthenticationConstants.BotFrameworkTokenIssuer },
+                ValidIssuers = new[] { AuthenticationConstants.ToBotFromChannelTokenIssuer },
```

Comment:
This looks like an unrelated fix?

Reply:
@johnataylor Correct. That's showing up in this PR due to the rebasing against master. That was part of a PR (that's already accepted + merged) that Dan did earlier.